### PR TITLE
chore(generator): Generators

### PR DESCRIPTION
**Pull request by dmiller9911**
Added generators for PatternFly 4 components and new packages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,39 +102,40 @@
...
     "build:docs": "yarn build:babel && lerna run postbuild && yarn build:tokens && lerna run build --scope=@patternfly/react-docs --stream",
-    "build:ts": "node ./scripts/copyTS.js",
     "build:tokens": "lerna run build --scope=@patternfly/react-tokens --stream",
+    "build:ts": "node ./scripts/copyTS.js",
     "clean": "yarn clean:build",
     "clean:build": "rm -rf packages/*/dist",
-    "start": "concurrently \"yarn storybook:run\" \"yarn storybook:openurl\"",
     "commit": "git-cz",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "generate": "yarn plop",
```
Comment (dmiller9911) [inline package.json]: This was the only one added.  I sorted the rest to make it easier to find scripts.

**Comment by priley86** [comment]
A couple things I noted so far... 

**PF4 Component generator**
Using the bash terminal/built in VS Code terminal, I do not see the PF4 component options when using arrow up/down/left/right:

screen shot 2018-06-25 at 11 15 58 am

**Package Generator**
When using this generator, I seem to the package is generator underneath `/scripts/packages` instead of `/packages`

Screen Shot 2018-06-25 at 11.23.08 AM

This is great to see this coming along! I may have a similar use case with generating templates or Redux code soon too!

**Comment by dmiller9911** [comment]
@priley86 Issues should be resolved now.

**Comment by priley86** [comment]
This looks like a good initial place to start.

I opened [440](https://github.com/patternfly/patternfly-react/issues/440) to note the additional generator for PF3. Can you add any notes there?

**Pull request merged**